megamek/src/megamek/common/Bay.java

```diff
@@ -259,6 +259,21 @@ public void load(Entity unit) throws IllegalArgumentException {
         }
         return loaded;
     }
+    
+    /**
+     * Generate a raw list of the Ids stored in troops. 
+     * Used by MHQ in cases where we can't get the entities via Game
+     *
+     * @return
+     */
+    public List<Integer> getLoadedUnitIds() {
```

Comment:
Yeah, I don't really like Vectors, either, they're pretty annoying now that we have better tools.